src/app/courses/courses-list-item/courses-list-item.component.spec.ts

```diff
@@ -1,25 +1,86 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, TestBed } from '@angular/core/testing';
 
 import { CoursesListItemComponent } from './courses-list-item.component';
+import { FaIconComponent } from '@fortawesome/angular-fontawesome';
+import { faClock } from '@fortawesome/free-regular-svg-icons';
+import { faCalendarAlt } from '@fortawesome/free-regular-svg-icons';
+import { faPen } from '@fortawesome/free-solid-svg-icons';
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { Component } from '@angular/core';
+import { By } from '@angular/platform-browser';
+import { CoursesListItem } from './courses-list-item-model';
 
 describe('CoursesListItemComponent', () => {
-  let component: CoursesListItemComponent;
-  let fixture: ComponentFixture<CoursesListItemComponent>;
+
+  @Component({
+    template: `
+      <vp-courses-list-item
+        [item]="courcesItem" (delete)="deleteCourseById($event)">
+      </vp-courses-list-item>`
+  })
+  class TestHostComponent {
+    courcesItem: CoursesListItem = {id: 11, title: 'Video course #11 Title', creationDate: '', duration: '', description: ''};
+    deleteCourseById(event: number) {
+      console.log('Deleting course by #id:' + event);
```

Comment:
you are building mock component here, no implementation required, you probably will spy on this method and won't pass call through, on top of that console in most cases prohibited so just remove it